### PR TITLE
Add Flexibility to Remove Use of AdminDB

### PR DESCRIPTION
**Pull request by kdiluca**
# Issue

We want to allow for the flexibility to disable the use of the admin db and to process admin information on the node.

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Generally use squash merge to rebase and clean comments before merging
 - [ ] Update the [changelog](CHANGELOG.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

**Comment by kdiluca** [comment]
Will be adding tests.

File: src/mjolnir/pbfgraphparser.cc [hunk 1]
```diff
@@ -1076,22 +1091,26 @@ struct graph_callback : public OSMPBF::Callback {
       if (it != tag_handlers_.end()) {
         it->second();
       }
-
+      if (tag_.first == "driving_side") {
```
Comment (gknisely) [inline src/mjolnir/pbfgraphparser.cc:1094]: shouldn't this be done in the handler section?

File: src/mjolnir/pbfgraphparser.cc [hunk 2]
```diff
@@ -1076,22 +1091,26 @@ struct graph_callback : public OSMPBF::Callback {
...
-               tag_.first.substr(0, 22) == "pedestrian:conditional" ||
-               tag_.first.substr(0, 15) == "hgv:conditional" ||
-               tag_.first.substr(0, 17) == "moped:conditional" ||
-               tag_.first.substr(0, 16) == "mofa:conditional" ||
-               tag_.first.substr(0, 15) == "psv:conditional" ||
-               tag_.first.substr(0, 16) == "taxi:conditional" ||
-               tag_.first.substr(0, 15) == "bus:conditional" ||
-               tag_.first.substr(0, 15) == "hov:conditional" ||
-               tag_.first.substr(0, 21) == "emergency:conditional") {
+      if (tag_.first.substr(0, 20) == "motorcar:conditional" ||
```
Comment (gknisely) [inline src/mjolnir/pbfgraphparser.cc:1100]: why did we delete the else?

File: src/mjolnir/pbfgraphparser.cc [hunk 3]
```diff
@@ -1398,9 +1417,6 @@ struct graph_callback : public OSMPBF::Callback {
         way_.set_speed(default_speed_);
     }
 
-    // default to drive on right.
```
Comment (gknisely) [inline src/mjolnir/pbfgraphparser.cc:1401]: Why was the default removed?

Comment (gknisely) [reply src/mjolnir/pbfgraphparser.cc:1401]: ignore this comment.  I see that it is set above.

**Review by gknisely — APPROVED** [review]

**Pull request merged**